### PR TITLE
Fix MakePath for windows.

### PR DESCRIPTION
**Pull request by cnv1989**
Thanks for contributing to DLR! By submitting this pull request, you confirm that your contribution is made under the terms of the [Apache 2.0 license](https://github.com/neo-ai/neo-ai-dlr/blob/master/LICENSE).

Please refer to our [guideline](https://github.com/neo-ai/neo-ai-dlr/blob/master/CONTRIBUTING.md) for useful information and tips.

### CODE REVIEW

File: tests/cpp/dlr_test.cc [hunk 1]
```diff
@@ -301,6 +302,45 @@ TEST(DLR, TestCreateFromPaths_RelayVM) {
...
 
+TEST(DLR, TestMakePathVecForSingleWindowsPath) {
+  std::string windows_path = "C:\\a\\b\\c:";
+  std::vector<std::string> path_vec = dlr::MakePathVec(windows_path);
+  EXPECT_EQ(path_vec.size(), 1);
+  EXPECT_EQ(path_vec[0], "C:\\a\\b\\c");
+}
+
+TEST(DLR, TestMakePathVecForMultipleWindowsPaths) {
+  std::string windows_path = "C:\\a\\b\\c:D:\\a\\b\\c";
```
Comment (mycpuorg) [inline tests/cpp/dlr_test.cc:313]: +1

**Review by mycpuorg — APPROVED** [review]

File: include/dlr_common.h [hunk 2]
```diff
@@ -113,6 +113,8 @@ std::string GetStringFromDeviceType(DLDeviceType device_type);
 
 DLDeviceType GetDeviceTypeFromMetadata(const std::vector<std::string>& model_paths);
 
+std::vector<std::string> MakePathVec(std::string model_path);
```
Comment (trevor-m) [inline include/dlr_common.h:116]: `std::string model_path` -> `const std::string& model_path`

File: tests/cpp/dlr_test.cc [hunk 3]
```diff
@@ -301,6 +302,44 @@ TEST(DLR, TestCreateFromPaths_RelayVM) {
...
+  std::vector<std::string> path_vec = dlr::MakePathVec(windows_path);
+  EXPECT_EQ(path_vec[0], "C:\\a\\b\\c");
+  EXPECT_EQ(path_vec[1], "D:\\a\\b\\c");
+}
+
+TEST(DLR, TestMakePathVecForSingleLinuxPath) {
+  std::string linux_path = "/a/b/c:";
```
Comment (trevor-m) [inline tests/cpp/dlr_test.cc:320]: Can you add a test for single linux path without `:` at end?

Ex: `std::string linux_path = "/a/b/c";`

**Review by trevor-m — CHANGES_REQUESTED** [review]

File: include/dlr_common.h (hunk 2 above)
Comment (cnv1989) [reply include/dlr_common.h:116]: this cannot be reference since we are passing char * as input.

**Review by trevor-m — APPROVED** [review]

**Pull request merged**